assets/js/ui/links.js

```diff
@@ -27,22 +27,29 @@
 'use strict';
 
 /**
- * Makes links that lead to other sites external.
+ * Makes content links that lead to other sites external.
  *
- * Adds the `external-link` class to every outbound link that is inside the `meetupContent`.
- * Also, adds `target="_blank"` attribute that opens markdown links in the new
- * browser tab.
+ * <p>Adds the `external-link` class to every outbound link that is on the `meetup`
+ * or `resources` page. Also, adds `target="_blank"` attribute that opens markdown
+ * links in the new browser tab.
  *
  * <p>The CSS will add a small arrow after the link.
  */
 export function makeLinksExternal() {
-    const content = document.querySelector('.meetup .resources');
-    const links = content.getElementsByTagName('a');
-    for (var i = 0, length = links.length; i < length; i++) {
-        var value = links[i].hostname && links[i].hostname !== location.hostname;
-        if (value) {
-            links[i].classList.add('external-link');
-            links[i].setAttribute('target', '_blank');
+    const isMeetupPage = document.body.classList.contains('meetup');
+    const isResourcesPage = document.body.classList.contains('resources');
+
+    if (isMeetupPage || isResourcesPage) {
```

Comment:
Try to find a solution without page checking. I don't like the idea of adding new checks for new pages.